app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/initialgainorloss/initial_gain_or_loss.scala.html

```diff
@@ -61,7 +61,7 @@
         content = Some(unitInput(
             fieldId = lossKey,
             value = form.data.getOrElse(lossKey, ""),
-            label = Html(messages(s"$lossKey.label")),
+            label = Html(""),
```

Comment:
I'm not sure about this, should it be empty string ?

Reply:
you are right. my bad :/ I've also added test cases to it.

Reply:
Done.